ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -148,7 +135,7 @@
     </li>
     <li class="item indicator" data-cy="search">
       <!-- svelte-ignore a11y-missing-attribute -->
-      <ConnectionStatusIndicator {peerState} />
+      <ConnectionStatusIndicator />
```

Comment:
I'd expect `ConnectionStatusIndicator`, a Design component, to receive the state and not to access the store itself, meaning here we would do:

``` svelte
<Remote {store} let:data={status}>
  <ConnectionStatusIndicator {status} />
</Remote>
```